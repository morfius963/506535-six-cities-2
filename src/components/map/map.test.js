import React from "react";
import {mount} from "enzyme";
import toJson from 'enzyme-to-json';
import Map from "./map.jsx";
import fixtureData from "../../__fixtures__/offers.js";

describe(`snapshot test`, () => {
  it(`Component correctly renders`, () => {
    const div = global.document.createElement(`div`);
    global.document.body.appendChild(div);

    const component = mount(<Map offers = {fixtureData} />, {attachTo: div});
    expect(toJson(component)).toMatchSnapshot();
  });
});
