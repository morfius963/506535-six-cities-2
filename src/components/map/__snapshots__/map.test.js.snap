// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test Component correctly renders 1`] = `
<Map
  offers={
    Array [
      Object {
        "isPremium": true,
        "location": Object {
          "coords": Array [
            52.3909553943508,
            4.85309666406198,
          ],
          "zoom": 12,
        },
        "previewImage": "img/apartment-02.jpg",
        "price": 250,
        "rating": 4.5,
        "title": "Beautiful &amp; luxurious apartment at great location",
        "type": "apartment",
      },
      Object {
        "isPremium": true,
        "location": Object {
          "coords": Array [
            52.369553943508,
            4.85309666406198,
          ],
          "zoom": 12,
        },
        "previewImage": "img/apartment-02.jpg",
        "price": 250,
        "rating": 4.5,
        "title": "Beautiful &amp; luxurious apartment at great location",
        "type": "apartment",
      },
      Object {
        "isPremium": true,
        "location": Object {
          "coords": Array [
            52.3909553943508,
            4.929309666406198,
          ],
          "zoom": 12,
        },
        "previewImage": "img/apartment-02.jpg",
        "price": 250,
        "rating": 4.5,
        "title": "Beautiful &amp; luxurious apartment at great location",
        "type": "apartment",
      },
      Object {
        "isPremium": true,
        "location": Object {
          "coords": Array [
            52.3809553943508,
            4.939309666406198,
          ],
          "zoom": 12,
        },
        "previewImage": "img/apartment-02.jpg",
        "price": 250,
        "rating": 4.5,
        "title": "Beautiful &amp; luxurious apartment at great location",
        "type": "apartment",
      },
    ]
  }
>
  <section
    className="cities__map map"
  >
    <div
      id="map"
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    />
  </section>
</Map>
`;
